### DISCUSSION TITLE
Implement auth(). getUserByEmail()

### DISCUSSION THREAD

**Discussion by jordynchuhaloff** [invertase/react-native-firebase#4501] [Ideas] [answered]
I don't believe this Firebase feature is supported by this lib yet, but it would be very helpful for us!

https://firebase.google.com/docs/auth/admin/manage-users#retrieve_user_data

**Comment by andersonaddo** [comment, marked as answer]
That's just an Admin SDK feature - it's server side only, not client.

**Reply by jordynchuhaloff** [reply]
Ah you're right
